src/confd/src/core.c

```diff
@@ -258,16 +258,21 @@ static int change_cb(sr_session_ctx_t *session, uint32_t sub_id, const char *mod
 
 		AUDIT("The new configuration has been applied.");
 	}
+
 free_diff:
-      lyd_free_tree(diff);
-      return rc;
+	lyd_free_tree(diff);
+	return rc;
 }
 
-static inline int subscribe_module(char *model, struct confd *confd, int flags) {
-	return sr_module_change_subscribe(confd->session, model, "//.", change_cb, confd,
-					  CB_PRIO_PRIMARY, SR_SUBSCR_CHANGE_ALL_MODULES | SR_SUBSCR_DEFAULT | flags, &confd->sub) &&
+static inline int subscribe_module(char *model, struct confd *confd, int flags)
```

Comment:
Since you already clean up my mess, rename the function to `subscribe_model`, that is more accurate